packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -47,10 +47,18 @@ import {
   VerifiedAgreementAttribute,
 } from "./models.js";
 
-type NotRevocableTenantAttribute = Pick<VerifiedTenantAttribute, "id">;
-type RevocableTenantAttribute =
-  | Pick<CertifiedTenantAttribute, "id" | "revocationTimestamp">
-  | Pick<DeclaredTenantAttribute, "id" | "revocationTimestamp">;
+const agreementCreationConflictingStates: AgreementState[] = [
```

Comment:
These 2 should also go in `packages/agreement-process/src/model/domain/models.ts` where you moved the other ones